src/main/java/de/bayern/gdi/gui/Controller.java

```diff
@@ -445,53 +479,81 @@ private void extractBoundingBox() {
             bbox += (envelope.getX() + envelope.getWidth()) + ",";
             bbox += (envelope.getY() + envelope.getHeight());
-            this.dataBean.addAttribute("bbox", bbox);
+            this.dataBean.addAttribute("bbox", bbox, "");
         } else {
             // Raise an error?
         }
     }
 
+
+    private boolean validateInput() {
+        String failed = "";
+        ArrayList<DataBean.Attribute> attributes
+                                = this.dataBean.getAttributes();
+        boolean ret = true;
+        Validator validator = Validator.getInstance();
+        for (DataBean.Attribute attribute: attributes) {
+            if (!attribute.type.equals("")) {
+                if (!validator.isValid(attribute.type, attribute.value)) {
+                    ret = false;
+                    if (failed.equals("")) {
+                        failed = attribute.name;
+                    } else {
+                        failed = failed + ", " + attribute.name;
+                    }
+                }
+            }
+        }
+        if (!failed.equals("")) {
+            statusBarText.setText(
+                    I18n.format("status.validation-fail", failed)
+            );
+        }
+        return ret;
+    }
+
     /**
      * Start the download.
      *
      * @param event The event
      */
     @FXML protected void handleDownload(ActionEvent event) {
-
-        DirectoryChooser dirChooser = new DirectoryChooser();
-        dirChooser.setTitle(I18n.getMsg("gui.save-dir"));
-        File selectedDir = dirChooser.showDialog(getPrimaryStage());
-        if (selectedDir == null) {
-            return;
-        }
-
         extractStoredQuery();
         extractBoundingBox();
-        this.dataBean.setProcessingSteps(extractProcessingSteps());
+        if (validateInput()) {
+            DirectoryChooser dirChooser = new DirectoryChooser();
+            dirChooser.setTitle(I18n.getMsg("gui.save-dir"));
+            File selectedDir = dirChooser.showDialog(getPrimaryStage());
+            if (selectedDir == null) {
+                return;
+            }
 
-        Task task = new Task() {
-            @Override
-            protected Integer call() {
-                String savePath = selectedDir.getPath();
-                DownloadStep ds = dataBean.convertToDownloadStep(savePath);
-                try {
-                    DownloadStepConverter dsc = new DownloadStepConverter(
-                        dataBean.getUserName(),
-                        dataBean.getPassword());
-                    JobList jl = dsc.convert(ds);
-                    Processor p = Processor.getInstance();
-                    p.addJob(jl);
-                } catch (final ConverterException ce) {
-                    Platform.runLater(() -> {
-                        statusBarText.setText(ce.getMessage());
-                    });
+            this.dataBean.setProcessingSteps(extractProcessingSteps());
+
+            Task task = new Task() {
```

Comment:
The Task is not needed here. The processor works already async.  But this can be fixed in a later change.